pkg/sensors/load.go

```diff
@@ -185,6 +189,13 @@ func (s *Sensor) LoadMaps(stopCtx context.Context, mapDir string) error {
 
 		pinPath := filepath.Join(mapDir, m.PinName)
 
+		if dir := filepath.Dir(m.PinName); isValidSubdir(dir) {
```

Comment:
Is it OK to not return an error if `isValidSubdir` fails, because the loading will fail?

Reply:
the condition there is to check if PinName has directory portion and create it,
dir will be '.' for PinName without subdirectory, so that's filtered out in isValidSubdir,
plus I added check for '..' which we don't want.. I'll add some comment